<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="EditOption">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>251</width>
      <height>431</height>
     </rect>
    </property>
    <property name="title">
     <string>User Adjustion</string>
    </property>
    <widget class="MyComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QSlider" name="ViewSlider">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>101</width>
       <height>29</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>181</width>
       <height>17</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Shading Model Selection:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>171</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Adjustment:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>81</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>View Scale:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>81</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Light-X:</string>
     </property>
    </widget>
    <widget class="QSlider" name="ViewSliderLight_X">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>101</width>
       <height>29</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>81</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Light-Y:</string>
     </property>
    </widget>
    <widget class="QSlider" name="ViewSliderLight_Y">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>101</width>
       <height>29</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>81</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Light-Z:</string>
     </property>
    </widget>
    <widget class="QSlider" name="ViewSliderLight_Z">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>101</width>
       <height>29</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_view">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>110</y>
       <width>48</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_LightX">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>140</y>
       <width>48</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_LightY">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>170</y>
       <width>48</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_LightZ">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>200</y>
       <width>48</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="MyComboBox" name="comboBox_pic_obj">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>181</width>
       <height>17</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Picture obj Selection:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>181</width>
       <height>17</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Picture src Selection:</string>
     </property>
    </widget>
    <widget class="MyComboBox" name="comboBox_pic_src">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>340</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="MyOpenGL" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>450</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Result Detail</string>
    </property>
    <widget class="QTextEdit" name="ShowTxt">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Current Shading Model</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyOpenGL</class>
   <extends>QOpenGLWidget</extends>
   <header>myopengl.h</header>
  </customwidget>
  <customwidget>
   <class>MyComboBox</class>
   <extends>QComboBox</extends>
   <header>mycombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
